### DISCUSSION TITLE
I am a beginner，May I ask if the mom6 model can simulate tidal processes in a region

### DISCUSSION THREAD

**Discussion by q1165600785** [mom-ocean/MOM6#1636] [General]
I don't seem to understand the difference between tidal simulation and ocean simulation

**Comment by marshallward** [comment]
Tidal forcing is one of the many forcings that can drive an ocean simulation.  So you can run an ocean simulation with or without tidal forcing.

Tidal forcing is critically important in regional simulations, but it can also play an important role in global simulations.

I suppose you might be asking about the generation of tidal forces from, for example, the Moon or Sun, but that does not happen in MOM6.  We use precomputed tables based on astronomic data.

There is much more to say of this topic, but it depends what you want to learn more about.

**Comment by marshallward** [comment]
I can't find any formal documentation on how to use tidal forcing, but here are some of the modules used to support tidal forcing or mixing.

* [MOM_tidal_forcing](https://mom6.readthedocs.io/en/main/api/generated/modules/mom_tidal_forcing.html)
* [MOM_tidal_mixing](https://mom6.readthedocs.io/en/main/api/generated/modules/mom_tidal_mixing.html)
* [MOM_int_tide_input](https://mom6.readthedocs.io/en/main/api/generated/modules/mom_int_tide_input.html)
* [MOM_internal_tides](https://mom6.readthedocs.io/en/main/api/generated/modules/mom_internal_tides.html)

**Reply by q1165600785** [reply]
> 我找不到有关如何使用潮汐强迫的任何正式文档，但这里有一些用于支持潮汐强迫或混合的模块。
> 
> * [MOM_潮汐力](https://mom6.readthedocs.io/en/main/api/generated/modules/mom_tidal_forcing.html)
> * [MOM_潮汐混合](https://mom6.readthedocs.io/en/main/api/generated/modules/mom_tidal_mixing.html)
> * [MOM_int_tide_input](https://mom6.readthedocs.io/en/main/api/generated/modules/mom_int_tide_input.html)
> * [MOM_internal_tides](https://mom6.readthedocs.io/en/main/api/generated/modules/mom_internal_tides.html)
I'm sorry for getting back to you so late. I studied the ADCirc model again a while ago, but the model is an unstructured grid. What I want to express is to use the TPOX tidal model as the boundary water level forcing condition to simulate tides. Can Momo6 do it

**Comment by awallcraft** [comment]
A paper on a tides-only MOM6 model configuration:
Improving Global Barotropic Tides With Sub-Grid Scale Topography
https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2023MS004056

A book chapter on tides in global 3-D ocean models:

2018	B.K. Arbic, M.H. Alford, J.K. Ansong, M.C. Buijsman, E.J. Metzger, H.E. Ngodock, J.G. Richman, J.F. Shriver, I. Souopgui, A.J. Wallcraft, L. Zamudio, et. al.	[A Primer on Global Internal Tide and Internal Gravity Wave Continuum Modeling in HYCOM and MITgcm](https://www7320.nrlssc.navy.mil/pubs/2018/shriver1-2018.pdf)

**Comment by aekiss** [comment]
The https://github.com/COSIMA/regional-mom6 package makes it much easier to set up regional MOM6 configurations, including with tidal forcing.

**Comment by q1165600785** [comment]
@marshallward   Professor, what are the output results of mom6 and where can I read them? Is there an introductory manual for mom6? I want to obtain the harmonic constants of tides. Which variable should I perform harmonic analysis on for the output

**Reply by marshallward** [reply]
Documentation is available at the MOM6-examples wiki, although it may not contain exactly what you need.

  https://github.com/NOAA-GFDL/MOM6-examples/wiki

Output diagnostics are defined by the `diag_table` file.  This file can be a bit cryptic, and I don't know if it's documented anywhere, but you can find many examples in the MOM6-examples repository.  The `available_diags` output file is also useful, but the available diagnostics may depend on your parameters in `MOM_input`.

I am not sure about specific instructions for tidal simulations, but perhaps others with more experience will be able to advise you.  I will check with our regional modelers and see if they have any resources available.
